docs/building.md

```diff
@@ -12,7 +12,7 @@ This part assumes you are not interested in building the Android or iOS wrappers
 The following command will ensure your environment variables are set properly and build the dependencies needed by the project:
 
 ```
-./libs/verify-desktop-environment.sh
+cargo verify_desktop
```

Comment:
-1 on that, I don't mind having a-sdev as an optional thing to the repo, however I disagree with making it mandatory (which is the de-facto the case when it replaces the canonical commands in docs)